### PR TITLE
Update plan-to-coexist-with-third-party-messaging-and-aad-ds.md

### PR DESCRIPTION
**Pull request by TransVaultCTO**
Updated Transvault wording to a lower case v. Expanded narrative to match that of other third party vendors.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/plan-to-coexist-with-third-party-messaging-and-aad-ds.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Here are some third-party migration tools and partners that can assist with Exch
 
 - **[Quest](https://www.quest.com/)**: Provider of on-premises and hosted migration and coexistence software, including pre-migration analysis and complete user and application coexistence. Full-featured migrations from on-premises Microsoft Exchange, IBM Domino, Novell GroupWise, Zimbra and other environments to Microsoft 365, Office 365, Exchange Online, and SharePoint Online.
 
-- **[Transvault](http://www.transvault.com/)**: Provider of migration solutions to Exchange Online.
+- **[Transvault](https://www.transvault.com/cloud-office-migrations/email-archive-migrations/)**: Provider of Cloud Office migration solutions to Microsoft 365 from Exchange and Notes. Transvault supports 23 different sources for migration and has products which deliver any size of project, complex email archive migrations and PST management. The enterprise migration solutions are secure, compliant, efficient and user-focused, and can be run both on-premises and in the Cloud.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/plan-to-coexist-with-third-party-messaging-and-aad-ds.md:204]:
```suggestion
- **[Transvault](https://www.transvault.com/cloud-office-migrations/email-archive-migrations/)**: Provider of cloud office migration solutions to Microsoft 365 from Microsoft Exchange and Notes. Transvault supports 23 different sources for migration and has products that deliver any size of project, complex email archive migrations, and PST management. The enterprise migration solutions are secure, compliant, efficient, and user-focused, and can be run both on-premises and in the cloud.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by JohanFreelancer9** [comment]
@TransVaultCTO - please commit my copy review suggestions. Many thanks.

**Comment by scanum** [comment]
@TransVaultCTO - Following up here.

**Comment by scanum** [comment]
@TransVaultCTO Please commit the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by scanum** [comment]
@TransVaultCTO Please commit the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**